base/applications/wordpad/lang/id-ID.rc

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright 2004 by Krzysztof Foltman
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ * 
+ * Translated by Reinhart Previano K. <reinhart@reinhart1010.id> 2022
+ */
+
+LANGUAGE LANG_INDONESIAN, SUBLANG_DEFAULT
+
+IDM_MAINMENU MENU
+BEGIN
+    POPUP "Be&rkas"
+    BEGIN
+        MENUITEM "Bar&u...\tCtrl+N",            ID_FILE_NEW
+        MENUITEM "&Buka...\tCtrl+O",            ID_FILE_OPEN
+        MENUITEM "&Simpan\tCtrl+S",             ID_FILE_SAVE
+        MENUITEM "Simpan seb&agai...",          ID_FILE_SAVEAS
+        MENUITEM SEPARATOR
+        MENUITEM "&Cetak...\tCtrl+P",           ID_PRINT
+        MENUITEM "P&ratinjau Cetakan...",       ID_PREVIEW
+        MENUITEM "&Penyetelan Halaman...",      ID_PRINTSETUP
+        MENUITEM SEPARATOR
+        MENUITEM "&Keluar",                     ID_FILE_EXIT
+    END
+    POPUP "&Edit"
+    BEGIN
+        MENUITEM "&Urungkan\tCtrl+Z",           ID_EDIT_UNDO
+        MENUITEM "U&langi\tCtrl+Y",             ID_EDIT_REDO
+        MENUITEM SEPARATOR
+        MENUITEM "Gu&nting\tCtrl+X",            ID_EDIT_CUT
+        MENUITEM "&Salin\tCtrl+C",              ID_EDIT_COPY
+        MENUITEM "&Tempel\tCtrl+V",             ID_EDIT_PASTE
+        MENUITEM "&Hapus\tDel",                 ID_EDIT_CLEAR
+        MENUITEM "Pilih semu&a\tCtrl+A",        ID_EDIT_SELECTALL
+        MENUITEM SEPARATOR
+        MENUITEM "Ca&ri...\tCtrl+F",            ID_FIND
+        MENUITEM "Cari B&erikutnya\tF3",        ID_FIND_NEXT
+        MENUITEM "&Ganti...\tCtrl+H",           ID_REPLACE
+        MENUITEM SEPARATOR
+        MENUITEM "Read-&only",                  ID_EDIT_READONLY
```

Comment:
Will it be translated?

Reply:
I intended to use the term as is as it is difficult to translate "read-only" here:
+ Closest translation is "hanya baca" while it has a different meaning, "only read *sth.*"
+ Another possibility would be "lindungi dokumen dari penyuntingan" which means "write-protect this document".

Reply:
How bout "baca-saja" ?. Some words can have multiple meanings. It's depend on reader POV or context. "Just read"  also can be meaned as "baru membaca" instead of "baca saja" or "hanya baca".

Reply:
```suggestion
        MENUITEM "Baca Sa&ja",                  ID_EDIT_READONLY
```